docs/development.md

```diff
@@ -100,18 +106,18 @@ To debug the code generator:
 
 ### Built in commands
 
-There are a number of custom rush commands to help with development. See the `.scripts` folder for more.
+There are a number of custom pnpm commands to help with development. See the `.scripts` folder for more. Add the `-r` switch for the script to apply to every package.
 
 To run `go build` and `go vet` on every generated module:
 
 ```terminal
-rush buildvet
+pnpm -r buildvet
```

Comment:
This was supposed to be `-w`, not `-r`. e.g. if you're working in `/packages/autorest.go` you'd do `pnpm -w modtidy $cwd`

Reply:
fixed in [44a86ad](https://github.com/Azure/autorest.go/pull/1552/commits/44a86adea428f9e8c59053a744cea5bbc303ba6e)